### PR TITLE
Master website sale stock link warehouse kic

### PR DESCRIPTION
**Pull request by kirti-c**
Description of the issue/feature this PR addresses:
Task ID - https://www.odoo.com/web#id=1913031&action=327&model=project.task&view_type=form&menu_id=4720
Pad ID - https://pad.odoo.com/p/r.bd614c793c7ec8a9861a997e07281c8c

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale_stock/models/website.py [hunk 1]
```diff
@@ -1,15 +1,19 @@
 # -*- coding: utf-8 -*-
-from odoo import api, models
+from odoo import api, fields, models
+
 
 class Website(models.Model):
     _inherit = 'website'
 
+    warehouse_id = fields.Many2one('stock.warehouse', string='Warehouse', default=lambda self: self.env.user.company_id)
```
Comment (JKE-be) [inline addons/website_sale_stock/models/website.py]: a default company_id for a warehouse_id will not work properly ;)

File: addons/website_sale_stock/models/website.py [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
-from odoo import api, models
+from odoo import api, fields, models
+
 
 class Website(models.Model):
     _inherit = 'website'
 
+    @api.model
+    def _default_warehouse_id(self):
+        company = self.env.user.company_id.id
```
Comment (JKE-be) [inline addons/website_sale_stock/models/website.py]: we should probably uses self.company_id no?

Comment (JKE-be) [inline addons/website_sale_stock/models/website.py]: we should probably uses self.company_id no?
what you think about?

**Comment by JKE-be** [comment]
And please, follow guidelines from yesterday training about the commit message :)
 https://docs.google.com/presentation/d/1wvVdIAKrPYWATlgbWwXHslJnf20TOMJd8SsC_RkuCME/view?usp=sharing

File: addons/website_sale_stock/models/website.py (hunk 2 above)
Comment (jir-odoo) [reply addons/website_sale_stock/models/website.py]: is it set before default?

Comment (JKE-be) [reply addons/website_sale_stock/models/website.py]: yes, you have right.
So dont make a default value.

btw, the code will fallback to the same logic in _prepare_sale_order_values if not default value is set.

I propose to just add domain on res config setting to restrict the warehouse to the current company.

**Comment by JKE-be** [comment]
@robodoo r+

**Pull request closed without merging**